apps/charterafrica/src/components/Research/Research.js

```diff
@@ -0,0 +1,29 @@
+import { Section } from "@commons-ui/core";
+import { Box, Divider, Grid } from "@mui/material";
+import { forwardRef } from "react";
+
+import PostCard from "../Postcard";
+
+import { secondary } from "@/charterafrica/colors";
+
+const Research = forwardRef((props, ref) => {
+  const { sx, research } = props;
+  return (
+    <Box bgcolor={secondary[50]} sx={sx} ref={ref}>
+      <Section sx={{ pl: { xs: 4, sm: 0 }, pr: { xs: 4, sm: 0 } }} ref={ref}>
```

Comment:
```suggestion
      <Section sx={{ px: { xs: "57.5px", sm: 0 } }}>
```

`px` = `pl` _and_ `pr`